src/core/tests/frontend/paddle/test_models/gen_scripts/generate_sum.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import print_function
+import sys
+import paddle
+import paddle.fluid as fluid
+
+from save_model import saveModel
+
+
+def sum_1(name: str, shape: list, dtype, values):
+    paddle.enable_static()
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        input = fluid.layers.fill_constant(shape=shape, dtype=dtype, value=values[0])
+        out = fluid.layers.sum(input)
+        cpu = paddle.static.cpu_places(1)
+        exe = paddle.static.Executor(cpu[0])
+        # startup program will call initializer to initialize the parameters.
+        exe.run(paddle.static.default_startup_program())
+        outs = exe.run(
+            fetch_list=[out])
+        saveModel(name, exe, feedkeys=[], fetchlist=[out], inputs=[], outputs=[outs[0]], target_dir=sys.argv[1])
+
+    return outs[0]
+
+
+def sum_2(name: str, shape: list, dtype, values):
+    paddle.enable_static()
+    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
+        input1 = fluid.layers.fill_constant(shape=shape, dtype=dtype, value=values[0])
+        input2 = fluid.layers.fill_constant(shape=shape, dtype=dtype, value=values[1])
+        input3 = fluid.layers.fill_constant(shape=shape, dtype=dtype, value=values[2])
+        out = fluid.layers.sum([input1, input2, input3])
+        cpu = paddle.static.cpu_places(1)
+        exe = paddle.static.Executor(cpu[0])
+        # startup program will call initializer to initialize the parameters.
+        exe.run(paddle.static.default_startup_program())
+        outs = exe.run(
+            fetch_list=[out])
+        saveModel(name, exe, feedkeys=[], fetchlist=[out], inputs=[], outputs=[outs[0]], target_dir=sys.argv[1])
```

Comment:
Can we combine sum1 and sum2 function into one? seems most parts of them are reusable.

Reply:
updated